screens/OrderDisplayScreen.js

```diff
@@ -67,6 +69,12 @@ class OrderDisplayScreen extends React.Component {
     this.getLabels()
   }
 
+  componentWillUnmount() {
```

Comment:
What does this do?

Reply:
I put this in all screens used socket. It's a try to fix warning of memory leak.